forgerock-auth/build.gradle

```diff
@@ -86,10 +87,15 @@ dependencies {
     compileOnly 'com.google.android.gms:play-services-safetynet:18.0.1'
     compileOnly 'net.openid:appauth:0.7.1'
     compileOnly 'com.google.android.gms:play-services-fido:18.1.0'
+    compileOnly 'com.google.android.gms:play-services-basement:18.1.0'
 
+    //For Device Binding
     compileOnly 'androidx.biometric:biometric-ktx:1.2.0-alpha04'
     compileOnly 'com.nimbusds:nimbus-jose-jwt:9.23'
-    compileOnly 'com.google.android.gms:play-services-basement:18.1.0'
+
+    //Application Pin
+    compileOnly 'com.madgag.spongycastle:bcpkix-jdk15on:1.58.0.0'
+    compileOnly 'androidx.security:security-crypto:1.0.0'
```

Comment:
we have a crypto art of forgerock-core, why dont we reuse it ?

Reply:
In forgerock-core, we define the crypto dependency as "implementation" but not "api", since it is not with "api" forgerock-auth does not have access to the crypto interface.
More information can be found here:
https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_separation